flake8_isort.py

```diff
@@ -1,4 +1,7 @@
 # -*- coding: utf-8 -*-
+from flake8_polyfill import stdin
+stdin.monkey_patch('pycodestyle')
```

Comment:
@carljm Is important the import order here? if not, please put the second line *after* all the imports